src/edge-core-index.js

```diff
@@ -514,7 +514,10 @@ export type EdgeCurrencyWallet = {
   signTx(tx: EdgeTransaction): Promise<EdgeTransaction>,
   broadcastTx(tx: EdgeTransaction): Promise<EdgeTransaction>,
   saveTx(tx: EdgeTransaction): Promise<void>,
-  sweepPrivateKey(keyUri: string): Promise<void>,
+  sweepPrivateKeys(
+    edgeSpendInfo: EdgeSpendInfo,
+    options?: any
```

Comment:
This needs to be typed. I'm going to approve this, even though it's rubbish, because we need to get this feature out the door.